web/package-lock.json

```diff
@@ -0,0 +1,2494 @@
+{
```

Comment:
I didn't realize this file got included in the original PR in #28 for some reason.  We should likely remove this.